rpc/transaction_test.go

```diff
@@ -208,14 +208,6 @@ func TestTransactionReceipt_MatchesCapturedTransaction(t *testing.T) {
 				Keys: []*felt.Felt{utils.TestHexToFelt(t, "0xf806f71b19e4744968b37e3fb288e61309ab33a782ea9d11e18f67a1fbb110")},
 			},
 		},
-		ExecutionResources: ExecutionResources{
-			Steps:          217182,
-			MemoryHoles:    6644,
-			PedersenApps:   2142,
-			RangeCheckApps: 8867,
-			BitwiseApps:    900,
-			ECDSAApps:      1,
-		},
```

Comment:
I don't understand why this is being removed. Should this not be replaced with `ComputationResources`?

Reply:
Same for others

Reply:
Re-added